### ISSUE TITLE
Add support for Typescript with React/React-Native

### ISSUE THREAD

**Issue by mathieubruguier** [Jaredk3nt/laserwave#27]
TS support with React/React-Native would be amazing.
It mostly works for now, but it looks like types appear white? 

![image](https://user-images.githubusercontent.com/792891/74890733-c9acae80-5339-11ea-9da7-3e3c86302918.png)

Here's a code sample from the [official React-Native website](https://facebook.github.io/react-native/docs/typescript):

``` ts
// components/Hello.tsx
import React from 'react';
import {Button, StyleSheet, Text, View} from 'react-native';

export interface Props {
    name: string;
    enthusiasmLevel?: number;
}

const Hello: React.FC<Props> = (props) => {
    const [enthusiasmLevel, setEnthusiasmLevel] = React.useState(props.enthusiasmLevel);

    const onIncrement = () => setEnthusiasmLevel((enthusiasmLevel || 0) + 1);
    const onDecrement = () => setEnthusiasmLevel((enthusiasmLevel || 0) - 1);

    const getExclamationMarks = (numChars: number) => Array(numChars + 1).join('!');
    return (
        <View style={styles.root}>
        <Text style={styles.greeting}>
            Hello{' '}
            {props.name + getExclamationMarks(enthusiasmLevel || 0)}
        </Text>

        <View style={styles.buttons}>
            <View style={styles.button}>
            <Button
                title="-"
                onPress={onDecrement}
                accessibilityLabel="decrement"
                color="red"
            />
            </View>

            <View style={styles.button}>
            <Button
                title="+"
                onPress={onIncrement}
                accessibilityLabel="increment"
                color="blue"
            />
            </View>
        </View>
        </View>
    );
    }
}

// styles
const styles = StyleSheet.create({
    root: {
        alignItems: 'center',
        alignSelf: 'center',
    },
    buttons: {
       flexDirection: 'row',
        minHeight: 70,
        alignItems: 'stretch',
        alignSelf: 'center',
        borderWidth: 5,
    },
    button: {
        flex: 1,
        paddingVertical: 0,
    },
    greeting: {
        color: '#999',
        fontWeight: 'bold',
    },
});
```

Thanks for your help!

**Comment by mathieubruguier** [comment]
I'm not 100% positive how to preview it inside VSCode? I'm sure it looks great haha

**Comment by Jaredk3nt** [comment]
Lost track of stuff, not sure if this was properly released, but it will be in v1.2.9 for sure 👍 sorry about that!
